include/open62541/server.h

```diff
@@ -1795,6 +1795,17 @@ typedef struct {
 UA_ServerStatistics UA_EXPORT
 UA_Server_getStatistics(UA_Server *server);
 
+/**
+* Localization
+* ------------
+*
+* A client may pass an array of locale IDs in the ActivateSession request (Part 4, 5.6.3.2).
+* This information can be used to return localized text to the client depending on its preferred
+* language(s).
+*/
+UA_EXPORT UA_THREADSAFE const UA_LocaleId *
+UA_Server_getLocaleIdsForSession(UA_Server *server, const UA_NodeId *sessionId, size_t *localeIdsSize);
```

Comment:
@jpfr The user needs access to this information to localize values in data source callbacks, so it can't be kept internal.
Our main use case for this PR is localizing messages and other properties in events.

Reply:
Ok, got it.

The method `UA_Server_getLocaleIdsForSession` is marked as thread-safe.
But it isn't.

Please make two versions of this. `UA_Server_getLocaleIdsForSession` which assumes that the session is stable as long as the data is used (and not marked as thread-safe).
And `UA_Server_getLocaleIdsCopyForSession` which generates a copy of the data and which is thread-safe.